### PR TITLE
(I18N) Chinese Simplified Translation Update

### PR DESCRIPTION
**Pull request by yangyangdaji**
# translation changes 
- [x] Translate `Invert Bookmarks` into `反转书签` and add comment (But my recommend is to translate it as `反向标注书签`)
- [x] `Debug日志`  > `调试日志`
- [x] `关闭当前` > `关闭当前文件`
- [x] `性质` > `属性`
- [x] Change `确定` into `OK` in update dialog
## optional changes
-  `̶书签`̶ ̶ ̶>̶ ̶`̶标签`̶ ̶ ̶(̶N̶e̶e̶d̶ ̶t̶o̶ ̶o̶n̶ ̶t̶h̶e̶ ̶s̶a̶m̶e̶ ̶p̶a̶g̶e̶)̶
- T̶r̶a̶n̶s̶l̶a̶t̶e̶ ̶`̶O̶p̶e̶n̶`̶ ̶i̶n̶t̶o̶ ̶`̶打开` i̶n̶
https://github.com/dail8859/NotepadNext/blob/b0ec36c1f6d9672dce0a505c06bbcdc6bcaae4f9/i18n/NotepadNext_zh_CN.ts#L124-L125
- [x] Maybe a lot more

Signed-off-by: yangyangdaji <1504305527@qq.com>

### CODE REVIEW

**Comment by yangyangdaji** [comment]
@dail8859 This PR will wait for 7 days , Waiting for review by others

**Comment by dail8859** [comment]
@yangyangdaji No problem at all. Just let me know when you are ready!

**Review by KangLin — APPROVED** [review]

**Comment by yangyangdaji** [comment]
https://github.com/dail8859/NotepadNext/blob/b0ec36c1f6d9672dce0a505c06bbcdc6bcaae4f9/i18n/NotepadNext_zh_CN.ts#L119-L125
Replace `Open` with `打开` in this case
Does anybody have a suggestion for that?

**Comment by yangyangdaji** [comment]
https://github.com/dail8859/NotepadNext/blob/b0ec36c1f6d9672dce0a505c06bbcdc6bcaae4f9/i18n/NotepadNext_zh_CN.ts#L110-L125
Translate `Open` into `打开` in this case
Does anybody have a suggestion for that?

**Comment by inkydragon** [comment]
> Translate Open into 打开 in this case
> Does anybody have a suggestion for that?

Just keep it consistent with the buttons in the actual UI.
Keep use english if the button is not translated.

(negative example) The update dialog for older versions like:
image

**Comment by yangyangdaji** [comment]
> (negative example) The update dialog for older versions like: image
oh I've got it

**Comment by yangyangdaji** [comment]
@inkydragon  I changed the `确定` to `OK` in update dialog
Now what does this PR look like

**Comment by yangyangdaji** [comment]
Ready to go  Waiting for Review again

**Review by inkydragon — APPROVED** [review]
LGTM

**Review by KangLin — APPROVED** [review]

**Comment by yangyangdaji** [comment]
@dail8859  This PR has been approved to wait for merging

**Comment by dail8859** [comment]
Thanks so much everyone!

**Pull request merged**
